lib/assets/javascripts/twine.coffee

```diff
@@ -1,11 +1,15 @@
 ((root, factory) ->
   if typeof root.define == 'function' && root.define.amd
-    root.define([], factory)
+    root.define(['jquery'], factory)
   else if typeof module == 'object' && module.exports
-    module.exports = factory()
+    jQuery = if typeof window != 'undefined'
+    then require('jquery')
+    else require('jquery')(root)
```

Comment:
I am assuming this is for SSR? ^

Reply:
Not sure what SSR stands for 😢?

When the `window` object isn't present, [jQuery returns a factory rather than the direct module](https://github.com/jquery/jquery/blob/master/src/wrapper.js#L21-L35).

Reply:
Oh sorry, I meant Server Side Rendering. 👍

Reply:
[Here's an example of the same tactic used in a UMD template](https://github.com/umdjs/umd/blob/master/templates/jqueryPlugin.js#L9-L24)